snuba/query/parser/expressions.py

```diff
@@ -28,7 +28,7 @@
 simple_term     = quoted_literal / numeric_literal / column_name
 literal         = ~r"[a-zA-Z0-9_\.:-]+"
 quoted_literal  = "'" string_literal "'"
-string_literal  = ~r"[a-zA-Z0-9_\.:-]+"
+string_literal  = ~r"[a-zA-Z0-9_\.:-]*"
```

Comment:
Didn't notice this earlier, but won't this disallow non-ASCII characters? I'm surprised we haven't see any errors around UTF-8 inputs (or maybe I am not looking in the right place?)